To: vim_dev@googlegroups.com
Subject: Patch 7.3.165
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.165
Problem:    ":find" completion does not escape spaces in a directory name.
	    (Isz)
Solution:   Add backslashes for EXPAND_FILES_IN_PATH. (Carlo Teubner)
Files:	    src/ex_getln.c


*** ../vim-7.3.164/src/ex_getln.c	2010-12-02 16:01:23.000000000 +0100
--- src/ex_getln.c	2011-04-28 12:52:12.000000000 +0200
***************
*** 3702,3707 ****
--- 3702,3708 ----
      if (options & WILD_ESCAPE)
      {
  	if (xp->xp_context == EXPAND_FILES
+ 		|| xp->xp_context == EXPAND_FILES_IN_PATH
  		|| xp->xp_context == EXPAND_SHELLCMD
  		|| xp->xp_context == EXPAND_BUFFERS
  		|| xp->xp_context == EXPAND_DIRECTORIES)
*** ../vim-7.3.164/src/version.c	2011-04-28 13:01:59.000000000 +0200
--- src/version.c	2011-04-28 17:17:53.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     165,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
38. You wake up at 3 a.m. to go to the bathroom and stop and check your e-mail
    on the way back to bed.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
